test/test_public_bindings.py

```diff
@@ -272,6 +272,663 @@ def test_no_new_bindings(self):
+            "torch.utils.data.communication.map",
+            "torch.utils.data.communication.messages",
+            "torch.utils.data.communication.protocol",
+            "torch.utils.data.communication.queue",
+            "torch.utils.data.dataloader",
+            "torch.utils.data.dataloader_experimental",
+            "torch.utils.data.datapipes",
+            "torch.utils.data.datapipes.dataframe",
+            "torch.utils.data.datapipes.dataframe.dataframe_wrapper",
+            "torch.utils.data.datapipes.dataframe.dataframes",
+            "torch.utils.data.datapipes.dataframe.datapipes",
+            "torch.utils.data.datapipes.dataframe.structures",
+            "torch.utils.data.datapipes.iter",
+            "torch.utils.data.datapipes.iter.callable",
+            "torch.utils.data.datapipes.iter.combinatorics",
+            "torch.utils.data.datapipes.iter.combining",
+            "torch.utils.data.datapipes.iter.filelister",
+            "torch.utils.data.datapipes.iter.fileopener",
+            "torch.utils.data.datapipes.iter.grouping",
+            "torch.utils.data.datapipes.iter.routeddecoder",
+            "torch.utils.data.datapipes.iter.selecting",
+            "torch.utils.data.datapipes.iter.streamreader",
+            "torch.utils.data.datapipes.iter.utils",
+            "torch.utils.data.datapipes.map",
+            "torch.utils.data.datapipes.map.callable",
+            "torch.utils.data.datapipes.map.combinatorics",
+            "torch.utils.data.datapipes.map.combining",
+            "torch.utils.data.datapipes.map.grouping",
+            "torch.utils.data.datapipes.map.utils",
+            "torch.utils.data.datapipes.utils",
+            "torch.utils.data.datapipes.utils.common",
+            "torch.utils.data.datapipes.utils.decoder",
+            "torch.utils.data.dataset",
+            "torch.utils.data.distributed",
+            "torch.utils.data.gen_pyi",
+            "torch.utils.data.graph",
+            "torch.utils.data.graph_settings",
+            "torch.utils.data.sampler",
+            "torch.utils.dlpack",
+            "torch.utils.ffi",
+            "torch.utils.file_baton",
+            "torch.utils.hipify",
+            "torch.utils.hipify.constants",
+            "torch.utils.hipify.cuda_to_hip_mappings",
+            "torch.utils.hipify.hipify_python",
+            "torch.utils.hipify.version",
+            "torch.utils.hooks",
+            "torch.utils.mkldnn",
+            "torch.utils.mobile_optimizer",
+            "torch.utils.model_dump",
+            "torch.utils.model_zoo",
+            "torch.utils.show_pickle",
+            "torch.utils.tensorboard",
+            "torch.utils.tensorboard.summary",
+            "torch.utils.tensorboard.writer",
+            "torch.utils.throughput_benchmark",
+            "torch.version"
+        }
+        for _, modname, ispkg in pkgutil.walk_packages(path=torch.__path__, prefix=torch.__name__ + '.'):
```

Comment:
I think you're missing "torch" here?
You can refactor the content of the loop inside a function and call that function as well from outside with `torch` directly.

Reply:
oh true -- will add it